ecs_deploy/ecs.py

```diff
@@ -591,3 +611,6 @@ class TaskPlacementError(EcsError):
 
 class UnknownTaskDefinitionError(EcsError):
     pass
+
```

Comment:
Please add the missing empty line here (PEP8)

Reply:
Fixed.